tools/reviewtool.py

```diff
@@ -94,21 +94,20 @@ def check_files(self, head_dir):
         return True
 
     @staticmethod
-    def git_tags(git_root):
-        output = subprocess.check_output(['git', 'tag'], cwd=git_root).decode()
-        return list(filter(lambda s: s, output.split('\n')))  # skip empty lines
+    def git_tags(repo):
+        return list(map(lambda tag: tag.name, repo.tags))
 
     @staticmethod
     def isfile(head_dir, filename):
         return os.path.isfile(os.path.join(head_dir, filename))
 
-    def check_basics(self, base_dir, head_dir, project, branch):
+    def check_basics(self, base_repo, head_repo, project, branch):
         print('Inspecting project %s, branch %s.' % (project, branch))
-
+        base_dir, head_dir = base_repo.working_dir, head_repo.working_dir
```

Comment:
__[Flake8]__

[F841] local variable 'base_dir' is assigned to but never used


<!-- Comment by [SideCI](https://sideci.com) -->